app/main.py

```diff
@@ -1 +1,49 @@
-#  write your code here
+class Animal:
+
+    def __init__(self, name: str,
+                 appetite: int,
+                 is_hungry: bool = True) -> None:
+        self.name = name
+        self.appetite = appetite
+        self.is_hungry = is_hungry
+
+    def print_name(self) -> None:
+        print(f"Hello, I'm {self.name}")
+
+    def feed(self):
+        if self.is_hungry:
+            print(f"Eating {self.appetite} food points...")
+            self.is_hungry = False
+            return self.appetite
+        return 0
+
+
+class Cat(Animal):
+
+    def __init__(self, name: str,
+                 is_hungry=True,
+                 appetite=3) -> None:
+        super().__init__(name, appetite, is_hungry)
+
+    @staticmethod
+    def catch_mouse() -> None:
+        print("The hunt began!")
+
+
+class Dog(Animal):
+
+    def __init__(self, name: str,
+                 appetite=7,
+                 is_hungry=True) -> None:
+        super().__init__(name, appetite, is_hungry)
+
+    @staticmethod
+    def bring_slippers() -> None:
+        print("The slippers delivered!")
+
+
+def feed_animals(animals: list) -> int:
+    sum_of_food = 0
+    for animal in animals:
+        sum_of_food += animal.feed()
+    return sum_of_food
```

Comment:
use function `sum`

Reply:
I thought it will be more "pythonic"

Reply:
you can write it in one line - it will be more pythonic as well